asv/commands/run.py

```diff
@@ -232,12 +236,15 @@ def run(cls, conf, range_spec=None, steps=None, date_period=None,
                                          branch in conf.branches]))
             except NoSuchNameError as exc:
                 raise util.UserError(f'Unknown branch {exc} in configuration')
-        elif range_spec == 'EXISTING':
+        elif range_spec == "EXISTING":
             commit_hashes = get_existing_hashes(conf.results_dir)
         elif range_spec == "NEW":
             # New commits on each configured branches
             old_commit_hashes = get_existing_hashes(conf.results_dir)
             commit_hashes = repo.get_new_branch_commits(conf.branches, old_commit_hashes)
+        elif range_spec == "TAGS":
+            # All tags on each configured branches
+            commit_hashes = list(reversed(repo.get_tags().keys()))
```

Comment:
```suggestion
            commit_hashes = reversed(list(repo.get_tags().keys()))
```

Should fix CI for 3.7, otherwise this looks good to me.